linkerd/admin/src/test/scala/io/buoyant/linkerd/admin/LinkerdAdminTest.scala

```diff
@@ -14,6 +14,7 @@ class BuoyantMainTest extends TwitterServer {
   def main() {}
 }
 
+// TODO: these tests are not getting run
```

Comment:
more specifically, code inside the `new BuoyantMainTest {` clause is not getting run. i tried to piece together when in git history this could have happened, may have been in a previous repo.